Bento/Views/BentoReusableView.swift

```diff
@@ -1,26 +1,71 @@
-protocol BentoReusableView: AnyObject {
-    var containedView: UIView? { get set }
-    var contentView: UIView { get }
+protocol BentoReusableView: NativeView {
+    var containedView: NativeView? { get set }
+    var contentView: NativeView { get }
     var component: AnyRenderable? { get set }
+    var storage: [StorageKey: Any] { get set }
+}
+
+struct StorageKey: Hashable {
+    let component: Any.Type
+    let identifier: ObjectIdentifier
+
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(ObjectIdentifier(component))
+        hasher.combine(identifier)
+    }
+
+    static func == (lhs: StorageKey, rhs: StorageKey) -> Bool {
+        return lhs.component == rhs.component && lhs.identifier == rhs.identifier
+    }
 }
 
 extension BentoReusableView {
-    func bind(_ component: AnyRenderable?) {
+    func bind(_ component: AnyRenderable) {
+        // NOTE: Unbind without removing view.
+        let oldComponent = unbindIfNeeded(removesView: false)
+
+        // Set the new component.
         self.component = component
-        if let component = component {
-            let renderingView: UIView
-
-            if let view = containedView, type(of: view) == component.viewType {
-                renderingView = view
-            } else {
-                renderingView = component.viewType.generate()
-                containedView = renderingView
-            }
 
-            component.render(in: renderingView)
+        let renderingView: UIView
+
+        if let view = containedView, oldComponent?.componentType == component.componentType {
+            renderingView = view
         } else {
-            containedView = nil
+            renderingView = component.viewType.generate()
+            containedView = renderingView
+        }
+
+        component.render(in: renderingView)
+        component.didMount(
+            to: renderingView,
+            storage: ViewStorage(componentType: component.componentType, view: self)
+        )
+    }
+
+    func unbindIfNeeded() {
+        unbindIfNeeded(removesView: true)
+    }
+
+    @discardableResult
+    private func unbindIfNeeded(removesView: Bool) -> AnyRenderable? {
+        // Notify the old component, and clear the view storage.
+        component.zip(with: containedView) {
+            $0.willUnmount(
+                from: $1,
+                storage: ViewStorage(componentType: $0.componentType, view: self)
+            )
         }
+        storage = [:]
+
+        let component = self.component
+        self.component = nil
+
+        if removesView {
+            self.containedView = nil
+        }
```

Comment:
We spotted a crash that seems likely to me related to this code

@anders: The layout engine is triggered during deinit, which seems to occasionally cause crashes.

`deinit` -> `BentoReusableView.unbindIfNeeded()` -> `BentoReusableView.unbindIfNeeded(removedView: true)` -> `containedView.didSet` -> `BentoReusableView.containerViewDidChange(from:, to: )` ends up triggering AutoLayout (`NSISEngine`) on `deinit` and crashes with the following demangled stacktrace:

```
    20  UIKitCore               -[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:] + 217
    21  UIKitCore               __45-[UIView(Hierarchy) _postMovedFromSuperview:]_block_invoke + 112
    22  UIKitCore               -[UIView(Hierarchy) _postMovedFromSuperview:] + 795
    23  UIKitCore               __UIViewWasRemovedFromSuperview + 172
    24  UIKitCore               -[UIView(Hierarchy) removeFromSuperview] + 493
    25  Bento                   (extension in Bento):Bento.BentoReusableView.containerViewDidChange(from: __C.UIView?, to: __C.UIView?) -> () + 1094
    26  Bento                   Bento.TableViewContainerCell.containedView.didset : __C.UIView? + 189
    27  Bento                   Bento.TableViewContainerCell.containedView.setter : __C.UIView? + 288
    28  Bento                   protocol witness for Bento.BentoReusableView.containedView.setter : __C.UIView? in conformance Bento.TableViewContainerCell : Bento.BentoReusableView in Bento + 9
    29  Bento                   (extension in Bento):Bento.BentoReusableView.(unbindIfNeeded in _C920B861C8A0418300F3C0EC0904FFD1)(removesView: Swift.Bool) -> Bento.AnyRenderable? + 847
    30  Bento                   (extension in Bento):Bento.BentoReusableView.unbindIfNeeded() + 72
    31  Bento                   Bento.TableViewContainerCell.__deallocating_deinit + 79
    32  Bento                   @objc Bento.TableViewContainerCell.__deallocating_deinit + 36
    33  CoreFoundation          -[__NSArrayM dealloc] + 159
    34  libobjc.A.dylib         _ZN11objc_object17sidetable_releaseEb + 202
    35  UIKitCore               -[UITableView .cxx_destruct] + 1561
```

---

Full Stacktrace available below:

`2019-06-04 12:47:19.678839+0200 xctest[81284:6069914] *** Assertion failure in void _removeFromEngineVarTable(NSISEngineVar)(), /BuildRoot/Library/Caches/com.apple.xbs/Sources/Foundation_Sim/Foundation-1570.15/Foundation/Layout.subproj/IncrementalSimplex/NSISEngine.m:1518`

<details>
<summary><tt>2019-06-04 12:47:19.687117+0200 xctest[81284:6069914] [general] Caught exception during autorelease pool drain NSInternalInconsistencyException: It wasn't possible to remove variable:<unknown var (bug!) with engine as delegate:0x7f8b31560940>{id: 30430} from engine <NSISEngine: 0x7f8b31591020>{ delegate:0x7f8b32abfeb0</tt></summary>

```
  EngineVars:
      0: objective{id: 30202} rowIndex:0
     21: UITableViewCellContentView:0x7f8b32abfeb0.minY{id: 30229} rowIndex:1073741868
     23: UITableViewCellContentView:0x7f8b32abfeb0.minX{id: 30231} rowIndex:6
     25: UITableViewCellContentView:0x7f8b32abfeb0.Height{id: 30233} rowIndex:1073741829
     27: UITableViewCellContentView:0x7f8b32abfeb0.Width{id: 30235} rowIndex:44
     47: UIStackView:0x7f8b31584f90.Height{id: 30255} colIndex:1073741824
     49: UIView:0x7f8b31581680.Width{id: 30257} colIndex:4
     50: UIStackView:0x7f8b31584f90.Width{id: 30258} rowIndex:7
     52: UIView:0x7f8b315818b0.Width{id: 30260} rowIndex:12
     54: UIView:0x7f8b3155ef40.Width{id: 30264} colIndex:2
     55: 0x7f8b3155f280.marker{id: 30263} colIndex:19
     56: UIView:0x7f8b3157e7c0.Width{id: 30268} rowIndex:13
     57: 0x7f8b31586170.marker{id: 30267} colIndex:20
     58: UIView:0x7f8b3157e7c0.Height{id: 30270} rowIndex:1073741834
     59: 0x7f8b31583800.marker{id: 30269} colIndex:1073741839
     82: UIView:0x7f8b315818b0.minX{id: 30297} rowIndex:17
    145: UIView:0x7f8b31581680.minX{id: 30365} rowIndex:29
    146: 0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364} colIndex:39
    147: UIView:0x7f8b3155ef40.minX{id: 30367} rowIndex:30
    148: 0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366} colIndex:40
    149: 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368} rowIndex:31
    150: 0x7f8b31567a60:UISV-canvas-fit.negError{id: 30369} colIndex:30
    151: _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.minX{id: 30371} rowIndex:32
    152: 0x7f8b31568790:UISV-distributing-edge.marker{id: 30370} colIndex:43
    153: _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373} colIndex:3
    154: 0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372} colIndex:44
    155: 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374} colIndex:1
    156: _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378} rowIndex:35
    157: 0x7f8b31569420:UISV-distributing-edge.marker{id: 30377} colIndex:45
    158: _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.Width{id: 30380} rowIndex:36
    159: 0x7f8b31569530:UISV-distributing-edge.marker{id: 30379} colIndex:46
    160: 0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381} colIndex:11
    161: 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385} colIndex:25
    162: UIView:0x7f8b31581680.minY{id: 30387} rowIndex:1073741859
    163: 0x7f8b31585720:UISV-canvas-connection.marker{id: 30386} colIndex:1073741867
    165: UIView:0x7f8b31581680.Height{id: 30389} rowIndex:1073741860
    166: 0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388} colIndex:1073741868
    167: UIView:0x7f8b315818b0.minY{id: 30391} rowIndex:1073741861
    168: UIView:0x7f8b315818b0.Height{id: 30392} rowIndex:1073741863
    169: 0x7f8b3156b7f0:UISV-alignment.marker{id: 30390} colIndex:1073741870
    170: UIView:0x7f8b3155ef40.minY{id: 30394} rowIndex:1073741862
    171: UIView:0x7f8b3155ef40.Height{id: 30395} rowIndex:1073741864
    172: 0x7f8b31566e80:UISV-alignment.marker{id: 30393} colIndex:1073741872
    173: 0x7f8b31567280:UISV-alignment.marker{id: 30396} colIndex:1073741859
    174: 0x7f8b31567460:UISV-alignment.marker{id: 30399} colIndex:1073741869
    181: 0x7f8b31576fb0:UIView-Encapsulated-Layout-Left.marker{id: 30491} colIndex:48
    182: 0x7f8b31560d80:UIView-Encapsulated-Layout-Top.marker{id: 30492} colIndex:1073741848
    183: 0x7f8b315799e0:UIView-Encapsulated-Layout-Width.marker{id: 30493} colIndex:0
    184: 0x7f8b31576f60:UIView-Encapsulated-Layout-Height.marker{id: 30496} colIndex:1073741850
    185: 0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426} colIndex:9
    186: 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427} rowIndex:37
    187: 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428} rowIndex:25
    189: <unknown var (bug!) with engine as delegate:0x7f8b31560940>{id: 30430} colIndex:29
    190: 0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}.brokenNegError{id: 30431} colIndex:47

  Rows:
      0: objective{id: 30202} == {objective 0x7f8b3158df60: <1100:6, 1000:44, 750:-2.77768e+06, 250:-2.77758e+06, 49:15> + <49:0.0138889>*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + <49:-5>*0x7f8b3155f280.marker{id: 30263, var: 55} + <49:2>*0x7f8b31567a60:UISV-canvas-fit.negError{id: 30369, var: 150} + <1100:-1>*0x7f8b31568790:UISV-distributing-edge.marker{id: 30370, var: 152} + <1100:-0.00555556>*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + <1100:1>*0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372, var: 154} + <1100:1>*0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374, var: 155} + <1100:-1>*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377, var: 157} + <1100:1>*0x7f8b31569530:UISV-distributing-edge.marker{id: 30379, var: 159} + <1100:-1>*0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381, var: 160} + <1100:1>*0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385, var: 161} + <1100:1, 49:-5>*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185} + <1100:1.49012e-07>*<unknown var (bug!) with engine as delegate:0x7f8b31560940>{id: 30430, var: 189} + <1100:2>*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}.brokenNegError{id: 30431, var: 190}}
      6: UITableViewCellContentView:0x7f8b32abfeb0.minX{id: 30231} == 0 + 360*0x7f8b31576fb0:UIView-Encapsulated-Layout-Left.marker{id: 30491, var: 181}
      7: UIStackView:0x7f8b31584f90.Width{id: 30258} == 5400 + 5*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + -1800*0x7f8b3155f280.marker{id: 30263, var: 55} + -1800*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185}
     12: UIView:0x7f8b315818b0.Width{id: 30260} == 5400 + -1*UIView:0x7f8b31581680.Width{id: 30257, var: 49} + 4*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + -1800*0x7f8b3155f280.marker{id: 30263, var: 55} + 360*0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364, var: 146} + -360*0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366, var: 148} + -360*0x7f8b31568790:UISV-distributing-edge.marker{id: 30370, var: 152} + -2*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + 360*0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372, var: 154} + -360*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377, var: 157} + 360*0x7f8b31569530:UISV-distributing-edge.marker{id: 30379, var: 159} + -360*0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381, var: 160} + -1800*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185} + 360*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}.brokenNegError{id: 30431, var: 190}
     13: UIView:0x7f8b3157e7c0.Width{id: 30268} == 2880 + 360*0x7f8b31586170.marker{id: 30267, var: 57}
     17: UIView:0x7f8b315818b0.minX{id: 30297} == 0 + 1*UIView:0x7f8b31581680.Width{id: 30257, var: 49} + -360*0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364, var: 146} + 360*0x7f8b31568790:UISV-distributing-edge.marker{id: 30370, var: 152} + 1*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + -360*0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372, var: 154}
     25: 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428} == 3 + -0.00277778*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + -1*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377, var: 157} + 1*0x7f8b31569530:UISV-distributing-edge.marker{id: 30379, var: 159} + -1*0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381, var: 160} + 1*0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385, var: 161} + 1*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}.brokenNegError{id: 30431, var: 190}
     29: UIView:0x7f8b31581680.minX{id: 30365} == 0 + -360*0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364, var: 146}
     30: UIView:0x7f8b3155ef40.minX{id: 30367} == 5400 + 4*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + -1800*0x7f8b3155f280.marker{id: 30263, var: 55} + -360*0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366, var: 148} + -1800*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185}
     31: 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368} == 15 + 0.0138889*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + -5*0x7f8b3155f280.marker{id: 30263, var: 55} + 1*0x7f8b31567a60:UISV-canvas-fit.negError{id: 30369, var: 150} + -5*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185}
     32: _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.minX{id: 30371} == 0 + 1*UIView:0x7f8b31581680.Width{id: 30257, var: 49} + -360*0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364, var: 146} + 360*0x7f8b31568790:UISV-distributing-edge.marker{id: 30370, var: 152}
     35: _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378} == 5400 + 4*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + -1800*0x7f8b3155f280.marker{id: 30263, var: 55} + -360*0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366, var: 148} + -1*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + 360*0x7f8b31569530:UISV-distributing-edge.marker{id: 30379, var: 159} + -360*0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381, var: 160} + -1800*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185}
     36: _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.Width{id: 30380} == 0 + 1*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + 360*0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381, var: 160}
     37: 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427} == 3 + -1*0x7f8b31568790:UISV-distributing-edge.marker{id: 30370, var: 152} + -0.00277778*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + 1*0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372, var: 154} + 1*0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374, var: 155}
     44: UITableViewCellContentView:0x7f8b32abfeb0.Width{id: 30235} == 135000 + 360*0x7f8b315799e0:UIView-Encapsulated-Layout-Width.marker{id: 30493, var: 183}
    1073741829: UITableViewCellContentView:0x7f8b32abfeb0.Height{id: 30233} == 33300 + 360*0x7f8b31576f60:UIView-Encapsulated-Layout-Height.marker{id: 30496, var: 184}
    1073741834: UIView:0x7f8b3157e7c0.Height{id: 30270} == 3600 + 360*0x7f8b31583800.marker{id: 30269, var: 59}
    1073741859: UIView:0x7f8b31581680.minY{id: 30387} == 0 + -360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163}
    1073741860: UIView:0x7f8b31581680.Height{id: 30389} == 0 + 1*UIStackView:0x7f8b31584f90.Height{id: 30255, var: 47} + 360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163} + -360*0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388, var: 166}
    1073741861: UIView:0x7f8b315818b0.minY{id: 30391} == 0 + -360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163} + -360*0x7f8b31567280:UISV-alignment.marker{id: 30396, var: 173}
    1073741862: UIView:0x7f8b3155ef40.minY{id: 30394} == 0 + -360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163} + -360*0x7f8b31567460:UISV-alignment.marker{id: 30399, var: 174}
    1073741863: UIView:0x7f8b315818b0.Height{id: 30392} == 0 + 1*UIStackView:0x7f8b31584f90.Height{id: 30255, var: 47} + 360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163} + -360*0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388, var: 166} + -360*0x7f8b3156b7f0:UISV-alignment.marker{id: 30390, var: 169} + 360*0x7f8b31567280:UISV-alignment.marker{id: 30396, var: 173}
    1073741864: UIView:0x7f8b3155ef40.Height{id: 30395} == 0 + 1*UIStackView:0x7f8b31584f90.Height{id: 30255, var: 47} + 360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163} + -360*0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388, var: 166} + -360*0x7f8b31566e80:UISV-alignment.marker{id: 30393, var: 172} + 360*0x7f8b31567460:UISV-alignment.marker{id: 30399, var: 174}
    1073741868: UITableViewCellContentView:0x7f8b32abfeb0.minY{id: 30229} == 0 + 360*0x7f8b31560d80:UIView-Encapsulated-Layout-Top.marker{id: 30492, var: 182}

  Columns:
      0: 0x7f8b315799e0:UIView-Encapsulated-Layout-Width.marker{id: 30493} == rowIndexes:(44) rowHeads:(UITableViewCellContentView:0x7f8b32abfeb0.Width{id: 30235})
      1: 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374} == rowIndexes:(0, 37) rowHeads:(objective{id: 30202}, 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427})
      2: UIView:0x7f8b3155ef40.Width{id: 30264} == rowIndexes:(0, 7, 12, 30, 31, 35) rowHeads:(objective{id: 30202}, UIStackView:0x7f8b31584f90.Width{id: 30258}, UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b3155ef40.minX{id: 30367}, 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378})
      3: _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373} == rowIndexes:(0, 37, 17, 25, 12, 35, 36) rowHeads:(objective{id: 30202}, 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427}, UIView:0x7f8b315818b0.minX{id: 30297}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428}, UIView:0x7f8b315818b0.Width{id: 30260}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.Width{id: 30380})
      4: UIView:0x7f8b31581680.Width{id: 30257} == rowIndexes:(12, 17, 32) rowHeads:(UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b315818b0.minX{id: 30297}, _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.minX{id: 30371})
      9: 0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426} == rowIndexes:(0, 7, 12, 30, 31, 35) rowHeads:(objective{id: 30202}, UIStackView:0x7f8b31584f90.Width{id: 30258}, UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b3155ef40.minX{id: 30367}, 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378})
     11: 0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381} == rowIndexes:(0, 12, 25, 35, 36) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.Width{id: 30380})
     19: 0x7f8b3155f280.marker{id: 30263} == rowIndexes:(0, 7, 12, 30, 31, 35) rowHeads:(objective{id: 30202}, UIStackView:0x7f8b31584f90.Width{id: 30258}, UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b3155ef40.minX{id: 30367}, 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378})
     20: 0x7f8b31586170.marker{id: 30267} == rowIndexes:(13) rowHeads:(UIView:0x7f8b3157e7c0.Width{id: 30268})
     25: 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385} == rowIndexes:(0, 25) rowHeads:(objective{id: 30202}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428})
     29: <unknown var (bug!) with engine as delegate:0x7f8b31560940>{id: 30430} == rowIndexes:(0) rowHeads:(objective{id: 30202})
     30: 0x7f8b31567a60:UISV-canvas-fit.negError{id: 30369} == rowIndexes:(0, 31) rowHeads:(objective{id: 30202}, 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368})
     39: 0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364} == rowIndexes:(12, 17, 29, 32) rowHeads:(UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b315818b0.minX{id: 30297}, UIView:0x7f8b31581680.minX{id: 30365}, _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.minX{id: 30371})
     40: 0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366} == rowIndexes:(12, 30, 35) rowHeads:(UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b3155ef40.minX{id: 30367}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378})
     43: 0x7f8b31568790:UISV-distributing-edge.marker{id: 30370} == rowIndexes:(0, 12, 17, 32, 37) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b315818b0.minX{id: 30297}, _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.minX{id: 30371}, 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427})
     44: 0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372} == rowIndexes:(0, 12, 17, 37) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b315818b0.minX{id: 30297}, 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427})
     45: 0x7f8b31569420:UISV-distributing-edge.marker{id: 30377} == rowIndexes:(0, 12, 25) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428})
     46: 0x7f8b31569530:UISV-distributing-edge.marker{id: 30379} == rowIndexes:(0, 12, 25, 35) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378})
     47: 0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}.brokenNegError{id: 30431} == rowIndexes:(0, 12, 25) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428})
     48: 0x7f8b31576fb0:UIView-Encapsulated-Layout-Left.marker{id: 30491} == rowIndexes:(6) rowHeads:(UITableViewCellContentView:0x7f8b32abfeb0.minX{id: 30231})
    1073741824: UIStackView:0x7f8b31584f90.Height{id: 30255} == rowIndexes:(1073741860, 1073741863, 1073741864) rowHeads:(UIView:0x7f8b31581680.Height{id: 30389}, UIView:0x7f8b315818b0.Height{id: 30392}, UIView:0x7f8b3155ef40.Height{id: 30395})
    1073741839: 0x7f8b31583800.marker{id: 30269} == rowIndexes:(1073741834) rowHeads:(UIView:0x7f8b3157e7c0.Height{id: 30270})
    1073741848: 0x7f8b31560d80:UIView-Encapsulated-Layout-Top.marker{id: 30492} == rowIndexes:(1073741868) rowHeads:(UITableViewCellContentView:0x7f8b32abfeb0.minY{id: 30229})
    1073741850: 0x7f8b31576f60:UIView-Encapsulated-Layout-Height.marker{id: 30496} == rowIndexes:(1073741829) rowHeads:(UITableViewCellContentView:0x7f8b32abfeb0.Height{id: 30233})
    1073741859: 0x7f8b31567280:UISV-alignment.marker{id: 30396} == rowIndexes:(1073741863, 1073741861) rowHeads:(UIView:0x7f8b315818b0.Height{id: 30392}, UIView:0x7f8b315818b0.minY{id: 30391})
    1073741867: 0x7f8b31585720:UISV-canvas-connection.marker{id: 30386} == rowIndexes:(1073741859, 1073741860, 1073741863, 1073741861, 1073741864, 1073741862) rowHeads:(UIView:0x7f8b31581680.minY{id: 30387}, UIView:0x7f8b31581680.Height{id: 30389}, UIView:0x7f8b315818b0.Height{id: 30392}, UIView:0x7f8b315818b0.minY{id: 30391}, UIView:0x7f8b3155ef40.Height{id: 30395}, UIView:0x7f8b3155ef40.minY{id: 30394})
    1073741868: 0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388} == rowIndexes:(1073741860, 1073741863, 1073741864) rowHeads:(UIView:0x7f8b31581680.Height{id: 30389}, UIView:0x7f8b315818b0.Height{id: 30392}, UIView:0x7f8b3155ef40.Height{id: 30395})
    1073741869: 0x7f8b31567460:UISV-alignment.marker{id: 30399} == rowIndexes:(1073741864, 1073741862) rowHeads:(UIView:0x7f8b3155ef40.Height{id: 30395}, UIView:0x7f8b3155ef40.minY{id: 30394})
    1073741870: 0x7f8b3156b7f0:UISV-alignment.marker{id: 30390} == rowIndexes:(1073741863) rowHeads:(UIView:0x7f8b315818b0.Height{id: 30392})
    1073741872: 0x7f8b31566e80:UISV-alignment.marker{id: 30393} == rowIndexes:(1073741864) rowHeads:(UIView:0x7f8b3155ef40.Height{id: 30395})

  Constraints:
    <NSAutoresizingMaskLayoutConstraint:0x7f8b31560d80 h=--& v=--& 'UIView-Encapsulated-Layout-Top' UITableViewCellContentView:0x7f8b32abfeb0.minY == 0   (active, names: '|':Bento.TableViewContainerCell:0x7f8b34055000'(extension in BabylonUI):...' )>      Marker:0x7f8b31560d80:UIView-Encapsulated-Layout-Top.marker{id: 30492}
    <NSAutoresizingMaskLayoutConstraint:0x7f8b31576fb0 h=--& v=--& 'UIView-Encapsulated-Layout-Left' UITableViewCellContentView:0x7f8b32abfeb0.minX == 0   (active, names: '|':Bento.TableViewContainerCell:0x7f8b34055000'(extension in BabylonUI):...' )>     Marker:0x7f8b31576fb0:UIView-Encapsulated-Layout-Left.marker{id: 30491}
    <NSLayoutConstraint:0x7f8b3155f280 UIView:0x7f8b3155ef40.width == 0.2*UIStackView:0x7f8b31584f90.width - 3   (active)>      Marker:0x7f8b3155f280.marker{id: 30263}
    <NSLayoutConstraint:0x7f8b31566e80 'UISV-alignment' UIView:0x7f8b31581680.bottom == UIView:0x7f8b3155ef40.bottom   (active)>        Marker:0x7f8b31566e80:UISV-alignment.marker{id: 30393}
    <NSLayoutConstraint:0x7f8b31567280 'UISV-alignment' UIView:0x7f8b31581680.top == UIView:0x7f8b315818b0.top   (active)>      Marker:0x7f8b31567280:UISV-alignment.marker{id: 30396}
    <NSLayoutConstraint:0x7f8b31567460 'UISV-alignment' UIView:0x7f8b31581680.top == UIView:0x7f8b3155ef40.top   (active)>      Marker:0x7f8b31567460:UISV-alignment.marker{id: 30399}
    <NSLayoutConstraint:0x7f8b315678b0 'UISV-canvas-connection' H:[UIView:0x7f8b3155ef40]-(0)-|   (active, names: '|':UIStackView:0x7f8b31584f90 )>     Marker:0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366}
    <NSLayoutConstraint:0x7f8b31567a60 'UISV-canvas-fit' UIStackView:0x7f8b31584f90.width == 0 priority:49   (active)>      Marker:0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368}
    <NSLayoutConstraint:0x7f8b31567ab0 'UISV-fill-equally' _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.width == _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.width   (active)>     Marker:0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381}
    <NSLayoutConstraint:0x7f8b31568790 'UISV-distributing-edge' H:[UIView:0x7f8b31581680]-(0)-[_UIOLAGapGuide:0x7f8b31568650'UISV-distributing']   (active)>        Marker:0x7f8b31568790:UISV-distributing-edge.marker{id: 30370}
    <NSLayoutConstraint:0x7f8b315688a0 'UISV-distributing-edge' _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.trailing == UIView:0x7f8b315818b0.leading   (active)>      Marker:0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372}
    <NSLayoutConstraint:0x7f8b315691c0 'UISV-spacing' H:[UIView:0x7f8b31581680]-(>=3)-[UIView:0x7f8b315818b0]   (active)>       Marker:0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}
    <NSLayoutConstraint:0x7f8b31569420 'UISV-distributing-edge' H:[UIView:0x7f8b315818b0]-(0)-[_UIOLAGapGuide:0x7f8b31569320'UISV-distributing']   (active)>        Marker:0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}
    <NSLayoutConstraint:0x7f8b31569530 'UISV-distributing-edge' _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.trailing == UIView:0x7f8b3155ef40.leading   (active)>      Marker:0x7f8b31569530:UISV-distributing-edge.marker{id: 30379}
    <NSLayoutConstraint:0x7f8b3156b450 'UISV-spacing' H:[UIView:0x7f8b315818b0]-(>=3)-[UIView:0x7f8b3155ef40]   (active)>       Marker:0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}
    <NSLayoutConstraint:0x7f8b3156b7f0 'UISV-alignment' UIView:0x7f8b31581680.bottom == UIView:0x7f8b315818b0.bottom   (active)>        Marker:0x7f8b3156b7f0:UISV-alignment.marker{id: 30390}
    <NSLayoutConstraint:0x7f8b3156bed0 'UISV-canvas-connection' V:[UIView:0x7f8b31581680]-(0)-|   (active, names: '|':UIStackView:0x7f8b31584f90 )>     Marker:0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388}
    <NSLayoutConstraint:0x7f8b31576f60 'UIView-Encapsulated-Layout-Height' UITableViewCellContentView:0x7f8b32abfeb0.height == 92.5   (active)>     Marker:0x7f8b31576f60:UIView-Encapsulated-Layout-Height.marker{id: 30496}
    <NSLayoutConstraint:0x7f8b315799e0 'UIView-Encapsulated-Layout-Width' UITableViewCellContentView:0x7f8b32abfeb0.width == 375   (active)>        Marker:0x7f8b315799e0:UIView-Encapsulated-Layout-Width.marker{id: 30493}
    <NSLayoutConstraint:0x7f8b3157d620 'UISV-canvas-connection' UIStackView:0x7f8b31584f90.leading == UIView:0x7f8b31581680.leading   (active)>     Marker:0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364}
    <NSLayoutConstraint:0x7f8b31583800 UIView:0x7f8b3157e7c0.height == 10   (active)>       Marker:0x7f8b31583800.marker{id: 30269}
    <NSLayoutConstraint:0x7f8b31585720 'UISV-canvas-connection' UIStackView:0x7f8b31584f90.top == UIView:0x7f8b31581680.top   (active)>     Marker:0x7f8b31585720:UISV-canvas-connection.marker{id: 30386}
    <NSLayoutConstraint:0x7f8b31586170 UIView:0x7f8b3157e7c0.width == 8   (active)>     Marker:0x7f8b31586170.marker{id: 30267}

  Statistics:
    23 rows. Variable counts:
          1 ->   8
          2 ->   3
          3 ->   3
          4 ->   3
          5 ->   3
          6 ->   1
          7 ->   1
         13 ->   1
    30 cols. Variable counts:
          1 ->   9
          2 ->   5
          3 ->   6
          4 ->   3
          5 ->   2
          6 ->   4
          7 ->   1
}. userInfo: (null)
```
</details>

<details>
<summary><tt>2019-06-04 12:47:19.701373+0200 xctest[81284:6069914] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'It wasn't possible to remove variable:<unknown var (bug!) with engine as delegate:0x7f8b31560940>{id: 30430} from engine <NSISEngine: 0x7f8b31591020>{ delegate:0x7f8b32abfeb0</tt></summary>

```
EngineVars:
      0: objective{id: 30202} rowIndex:0
     21: UITableViewCellContentView:0x7f8b32abfeb0.minY{id: 30229} rowIndex:1073741868
     23: UITableViewCellContentView:0x7f8b32abfeb0.minX{id: 30231} rowIndex:6
     25: UITableViewCellContentView:0x7f8b32abfeb0.Height{id: 30233} rowIndex:1073741829
     27: UITableViewCellContentView:0x7f8b32abfeb0.Width{id: 30235} rowIndex:44
     47: UIStackView:0x7f8b31584f90.Height{id: 30255} colIndex:1073741824
     49: UIView:0x7f8b31581680.Width{id: 30257} colIndex:4
     50: UIStackView:0x7f8b31584f90.Width{id: 30258} rowIndex:7
     52: UIView:0x7f8b315818b0.Width{id: 30260} rowIndex:12
     54: UIView:0x7f8b3155ef40.Width{id: 30264} colIndex:2
     55: 0x7f8b3155f280.marker{id: 30263} colIndex:19
     56: UIView:0x7f8b3157e7c0.Width{id: 30268} rowIndex:13
     57: 0x7f8b31586170.marker{id: 30267} colIndex:20
     58: UIView:0x7f8b3157e7c0.Height{id: 30270} rowIndex:1073741834
     59: 0x7f8b31583800.marker{id: 30269} colIndex:1073741839
     82: UIView:0x7f8b315818b0.minX{id: 30297} rowIndex:17
    145: UIView:0x7f8b31581680.minX{id: 30365} rowIndex:29
    146: 0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364} colIndex:39
    147: UIView:0x7f8b3155ef40.minX{id: 30367} rowIndex:30
    148: 0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366} colIndex:40
    149: 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368} rowIndex:31
    150: 0x7f8b31567a60:UISV-canvas-fit.negError{id: 30369} colIndex:30
    151: _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.minX{id: 30371} rowIndex:32
    152: 0x7f8b31568790:UISV-distributing-edge.marker{id: 30370} colIndex:43
    153: _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373} colIndex:3
    154: 0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372} colIndex:44
    155: 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374} colIndex:1
    156: _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378} rowIndex:35
    157: 0x7f8b31569420:UISV-distributing-edge.marker{id: 30377} colIndex:45
    158: _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.Width{id: 30380} rowIndex:36
    159: 0x7f8b31569530:UISV-distributing-edge.marker{id: 30379} colIndex:46
    160: 0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381} colIndex:11
    161: 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385} colIndex:25
    162: UIView:0x7f8b31581680.minY{id: 30387} rowIndex:1073741859
    163: 0x7f8b31585720:UISV-canvas-connection.marker{id: 30386} colIndex:1073741867
    165: UIView:0x7f8b31581680.Height{id: 30389} rowIndex:1073741860
    166: 0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388} colIndex:1073741868
    167: UIView:0x7f8b315818b0.minY{id: 30391} rowIndex:1073741861
    168: UIView:0x7f8b315818b0.Height{id: 30392} rowIndex:1073741863
    169: 0x7f8b3156b7f0:UISV-alignment.marker{id: 30390} colIndex:1073741870
    170: UIView:0x7f8b3155ef40.minY{id: 30394} rowIndex:1073741862
    171: UIView:0x7f8b3155ef40.Height{id: 30395} rowIndex:1073741864
    172: 0x7f8b31566e80:UISV-alignment.marker{id: 30393} colIndex:1073741872
    173: 0x7f8b31567280:UISV-alignment.marker{id: 30396} colIndex:1073741859
    174: 0x7f8b31567460:UISV-alignment.marker{id: 30399} colIndex:1073741869
    181: 0x7f8b31576fb0:UIView-Encapsulated-Layout-Left.marker{id: 30491} colIndex:48
    182: 0x7f8b31560d80:UIView-Encapsulated-Layout-Top.marker{id: 30492} colIndex:1073741848
    183: 0x7f8b315799e0:UIView-Encapsulated-Layout-Width.marker{id: 30493} colIndex:0
    184: 0x7f8b31576f60:UIView-Encapsulated-Layout-Height.marker{id: 30496} colIndex:1073741850
    185: 0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426} colIndex:9
    186: 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427} rowIndex:37
    187: 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428} rowIndex:25
    189: <unknown var (bug!) with engine as delegate:0x7f8b31560940>{id: 30430} colIndex:29
    190: 0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}.brokenNegError{id: 30431} colIndex:47

  Rows:
      0: objective{id: 30202} == {objective 0x7f8b3158df60: <1100:6, 1000:44, 750:-2.77768e+06, 250:-2.77758e+06, 49:15> + <49:0.0138889>*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + <49:-5>*0x7f8b3155f280.marker{id: 30263, var: 55} + <49:2>*0x7f8b31567a60:UISV-canvas-fit.negError{id: 30369, var: 150} + <1100:-1>*0x7f8b31568790:UISV-distributing-edge.marker{id: 30370, var: 152} + <1100:-0.00555556>*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + <1100:1>*0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372, var: 154} + <1100:1>*0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374, var: 155} + <1100:-1>*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377, var: 157} + <1100:1>*0x7f8b31569530:UISV-distributing-edge.marker{id: 30379, var: 159} + <1100:-1>*0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381, var: 160} + <1100:1>*0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385, var: 161} + <1100:1, 49:-5>*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185} + <1100:1.49012e-07>*<unknown var (bug!) with engine as delegate:0x7f8b31560940>{id: 30430, var: 189} + <1100:2>*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}.brokenNegError{id: 30431, var: 190}}
      6: UITableViewCellContentView:0x7f8b32abfeb0.minX{id: 30231} == 0 + 360*0x7f8b31576fb0:UIView-Encapsulated-Layout-Left.marker{id: 30491, var: 181}
      7: UIStackView:0x7f8b31584f90.Width{id: 30258} == 5400 + 5*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + -1800*0x7f8b3155f280.marker{id: 30263, var: 55} + -1800*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185}
     12: UIView:0x7f8b315818b0.Width{id: 30260} == 5400 + -1*UIView:0x7f8b31581680.Width{id: 30257, var: 49} + 4*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + -1800*0x7f8b3155f280.marker{id: 30263, var: 55} + 360*0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364, var: 146} + -360*0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366, var: 148} + -360*0x7f8b31568790:UISV-distributing-edge.marker{id: 30370, var: 152} + -2*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + 360*0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372, var: 154} + -360*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377, var: 157} + 360*0x7f8b31569530:UISV-distributing-edge.marker{id: 30379, var: 159} + -360*0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381, var: 160} + -1800*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185} + 360*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}.brokenNegError{id: 30431, var: 190}
     13: UIView:0x7f8b3157e7c0.Width{id: 30268} == 2880 + 360*0x7f8b31586170.marker{id: 30267, var: 57}
     17: UIView:0x7f8b315818b0.minX{id: 30297} == 0 + 1*UIView:0x7f8b31581680.Width{id: 30257, var: 49} + -360*0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364, var: 146} + 360*0x7f8b31568790:UISV-distributing-edge.marker{id: 30370, var: 152} + 1*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + -360*0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372, var: 154}
     25: 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428} == 3 + -0.00277778*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + -1*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377, var: 157} + 1*0x7f8b31569530:UISV-distributing-edge.marker{id: 30379, var: 159} + -1*0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381, var: 160} + 1*0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385, var: 161} + 1*0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}.brokenNegError{id: 30431, var: 190}
     29: UIView:0x7f8b31581680.minX{id: 30365} == 0 + -360*0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364, var: 146}
     30: UIView:0x7f8b3155ef40.minX{id: 30367} == 5400 + 4*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + -1800*0x7f8b3155f280.marker{id: 30263, var: 55} + -360*0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366, var: 148} + -1800*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185}
     31: 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368} == 15 + 0.0138889*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + -5*0x7f8b3155f280.marker{id: 30263, var: 55} + 1*0x7f8b31567a60:UISV-canvas-fit.negError{id: 30369, var: 150} + -5*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185}
     32: _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.minX{id: 30371} == 0 + 1*UIView:0x7f8b31581680.Width{id: 30257, var: 49} + -360*0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364, var: 146} + 360*0x7f8b31568790:UISV-distributing-edge.marker{id: 30370, var: 152}
     35: _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378} == 5400 + 4*UIView:0x7f8b3155ef40.Width{id: 30264, var: 54} + -1800*0x7f8b3155f280.marker{id: 30263, var: 55} + -360*0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366, var: 148} + -1*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + 360*0x7f8b31569530:UISV-distributing-edge.marker{id: 30379, var: 159} + -360*0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381, var: 160} + -1800*0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426, var: 185}
     36: _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.Width{id: 30380} == 0 + 1*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + 360*0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381, var: 160}
     37: 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427} == 3 + -1*0x7f8b31568790:UISV-distributing-edge.marker{id: 30370, var: 152} + -0.00277778*_UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373, var: 153} + 1*0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372, var: 154} + 1*0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374, var: 155}
     44: UITableViewCellContentView:0x7f8b32abfeb0.Width{id: 30235} == 135000 + 360*0x7f8b315799e0:UIView-Encapsulated-Layout-Width.marker{id: 30493, var: 183}
    1073741829: UITableViewCellContentView:0x7f8b32abfeb0.Height{id: 30233} == 33300 + 360*0x7f8b31576f60:UIView-Encapsulated-Layout-Height.marker{id: 30496, var: 184}
    1073741834: UIView:0x7f8b3157e7c0.Height{id: 30270} == 3600 + 360*0x7f8b31583800.marker{id: 30269, var: 59}
    1073741859: UIView:0x7f8b31581680.minY{id: 30387} == 0 + -360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163}
    1073741860: UIView:0x7f8b31581680.Height{id: 30389} == 0 + 1*UIStackView:0x7f8b31584f90.Height{id: 30255, var: 47} + 360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163} + -360*0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388, var: 166}
    1073741861: UIView:0x7f8b315818b0.minY{id: 30391} == 0 + -360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163} + -360*0x7f8b31567280:UISV-alignment.marker{id: 30396, var: 173}
    1073741862: UIView:0x7f8b3155ef40.minY{id: 30394} == 0 + -360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163} + -360*0x7f8b31567460:UISV-alignment.marker{id: 30399, var: 174}
    1073741863: UIView:0x7f8b315818b0.Height{id: 30392} == 0 + 1*UIStackView:0x7f8b31584f90.Height{id: 30255, var: 47} + 360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163} + -360*0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388, var: 166} + -360*0x7f8b3156b7f0:UISV-alignment.marker{id: 30390, var: 169} + 360*0x7f8b31567280:UISV-alignment.marker{id: 30396, var: 173}
    1073741864: UIView:0x7f8b3155ef40.Height{id: 30395} == 0 + 1*UIStackView:0x7f8b31584f90.Height{id: 30255, var: 47} + 360*0x7f8b31585720:UISV-canvas-connection.marker{id: 30386, var: 163} + -360*0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388, var: 166} + -360*0x7f8b31566e80:UISV-alignment.marker{id: 30393, var: 172} + 360*0x7f8b31567460:UISV-alignment.marker{id: 30399, var: 174}
    1073741868: UITableViewCellContentView:0x7f8b32abfeb0.minY{id: 30229} == 0 + 360*0x7f8b31560d80:UIView-Encapsulated-Layout-Top.marker{id: 30492, var: 182}

  Columns:
      0: 0x7f8b315799e0:UIView-Encapsulated-Layout-Width.marker{id: 30493} == rowIndexes:(44) rowHeads:(UITableViewCellContentView:0x7f8b32abfeb0.Width{id: 30235})
      1: 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374} == rowIndexes:(0, 37) rowHeads:(objective{id: 30202}, 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427})
      2: UIView:0x7f8b3155ef40.Width{id: 30264} == rowIndexes:(0, 7, 12, 30, 31, 35) rowHeads:(objective{id: 30202}, UIStackView:0x7f8b31584f90.Width{id: 30258}, UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b3155ef40.minX{id: 30367}, 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378})
      3: _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.Width{id: 30373} == rowIndexes:(0, 37, 17, 25, 12, 35, 36) rowHeads:(objective{id: 30202}, 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427}, UIView:0x7f8b315818b0.minX{id: 30297}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428}, UIView:0x7f8b315818b0.Width{id: 30260}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.Width{id: 30380})
      4: UIView:0x7f8b31581680.Width{id: 30257} == rowIndexes:(12, 17, 32) rowHeads:(UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b315818b0.minX{id: 30297}, _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.minX{id: 30371})
      9: 0x7f8b3155f280.marker{id: 30263}.brokenPosError{id: 30426} == rowIndexes:(0, 7, 12, 30, 31, 35) rowHeads:(objective{id: 30202}, UIStackView:0x7f8b31584f90.Width{id: 30258}, UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b3155ef40.minX{id: 30367}, 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378})
     11: 0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381} == rowIndexes:(0, 12, 25, 35, 36) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.Width{id: 30380})
     19: 0x7f8b3155f280.marker{id: 30263} == rowIndexes:(0, 7, 12, 30, 31, 35) rowHeads:(objective{id: 30202}, UIStackView:0x7f8b31584f90.Width{id: 30258}, UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b3155ef40.minX{id: 30367}, 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378})
     20: 0x7f8b31586170.marker{id: 30267} == rowIndexes:(13) rowHeads:(UIView:0x7f8b3157e7c0.Width{id: 30268})
     25: 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385} == rowIndexes:(0, 25) rowHeads:(objective{id: 30202}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428})
     29: <unknown var (bug!) with engine as delegate:0x7f8b31560940>{id: 30430} == rowIndexes:(0) rowHeads:(objective{id: 30202})
     30: 0x7f8b31567a60:UISV-canvas-fit.negError{id: 30369} == rowIndexes:(0, 31) rowHeads:(objective{id: 30202}, 0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368})
     39: 0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364} == rowIndexes:(12, 17, 29, 32) rowHeads:(UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b315818b0.minX{id: 30297}, UIView:0x7f8b31581680.minX{id: 30365}, _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.minX{id: 30371})
     40: 0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366} == rowIndexes:(12, 30, 35) rowHeads:(UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b3155ef40.minX{id: 30367}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378})
     43: 0x7f8b31568790:UISV-distributing-edge.marker{id: 30370} == rowIndexes:(0, 12, 17, 32, 37) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b315818b0.minX{id: 30297}, _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.minX{id: 30371}, 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427})
     44: 0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372} == rowIndexes:(0, 12, 17, 37) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, UIView:0x7f8b315818b0.minX{id: 30297}, 0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}.brokenNegError{id: 30427})
     45: 0x7f8b31569420:UISV-distributing-edge.marker{id: 30377} == rowIndexes:(0, 12, 25) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428})
     46: 0x7f8b31569530:UISV-distributing-edge.marker{id: 30379} == rowIndexes:(0, 12, 25, 35) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428}, _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.minX{id: 30378})
     47: 0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}.brokenNegError{id: 30431} == rowIndexes:(0, 12, 25) rowHeads:(objective{id: 30202}, UIView:0x7f8b315818b0.Width{id: 30260}, 0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}.brokenNegError{id: 30428})
     48: 0x7f8b31576fb0:UIView-Encapsulated-Layout-Left.marker{id: 30491} == rowIndexes:(6) rowHeads:(UITableViewCellContentView:0x7f8b32abfeb0.minX{id: 30231})
    1073741824: UIStackView:0x7f8b31584f90.Height{id: 30255} == rowIndexes:(1073741860, 1073741863, 1073741864) rowHeads:(UIView:0x7f8b31581680.Height{id: 30389}, UIView:0x7f8b315818b0.Height{id: 30392}, UIView:0x7f8b3155ef40.Height{id: 30395})
    1073741839: 0x7f8b31583800.marker{id: 30269} == rowIndexes:(1073741834) rowHeads:(UIView:0x7f8b3157e7c0.Height{id: 30270})
    1073741848: 0x7f8b31560d80:UIView-Encapsulated-Layout-Top.marker{id: 30492} == rowIndexes:(1073741868) rowHeads:(UITableViewCellContentView:0x7f8b32abfeb0.minY{id: 30229})
    1073741850: 0x7f8b31576f60:UIView-Encapsulated-Layout-Height.marker{id: 30496} == rowIndexes:(1073741829) rowHeads:(UITableViewCellContentView:0x7f8b32abfeb0.Height{id: 30233})
    1073741859: 0x7f8b31567280:UISV-alignment.marker{id: 30396} == rowIndexes:(1073741863, 1073741861) rowHeads:(UIView:0x7f8b315818b0.Height{id: 30392}, UIView:0x7f8b315818b0.minY{id: 30391})
    1073741867: 0x7f8b31585720:UISV-canvas-connection.marker{id: 30386} == rowIndexes:(1073741859, 1073741860, 1073741863, 1073741861, 1073741864, 1073741862) rowHeads:(UIView:0x7f8b31581680.minY{id: 30387}, UIView:0x7f8b31581680.Height{id: 30389}, UIView:0x7f8b315818b0.Height{id: 30392}, UIView:0x7f8b315818b0.minY{id: 30391}, UIView:0x7f8b3155ef40.Height{id: 30395}, UIView:0x7f8b3155ef40.minY{id: 30394})
    1073741868: 0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388} == rowIndexes:(1073741860, 1073741863, 1073741864) rowHeads:(UIView:0x7f8b31581680.Height{id: 30389}, UIView:0x7f8b315818b0.Height{id: 30392}, UIView:0x7f8b3155ef40.Height{id: 30395})
    1073741869: 0x7f8b31567460:UISV-alignment.marker{id: 30399} == rowIndexes:(1073741864, 1073741862) rowHeads:(UIView:0x7f8b3155ef40.Height{id: 30395}, UIView:0x7f8b3155ef40.minY{id: 30394})
    1073741870: 0x7f8b3156b7f0:UISV-alignment.marker{id: 30390} == rowIndexes:(1073741863) rowHeads:(UIView:0x7f8b315818b0.Height{id: 30392})
    1073741872: 0x7f8b31566e80:UISV-alignment.marker{id: 30393} == rowIndexes:(1073741864) rowHeads:(UIView:0x7f8b3155ef40.Height{id: 30395})

  Constraints:
    <NSAutoresizingMaskLayoutConstraint:0x7f8b31560d80 h=--& v=--& 'UIView-Encapsulated-Layout-Top' UITableViewCellContentView:0x7f8b32abfeb0.minY == 0   (active, names: '|':Bento.TableViewContainerCell:0x7f8b34055000'(extension in BabylonUI):...' )>      Marker:0x7f8b31560d80:UIView-Encapsulated-Layout-Top.marker{id: 30492}
    <NSAutoresizingMaskLayoutConstraint:0x7f8b31576fb0 h=--& v=--& 'UIView-Encapsulated-Layout-Left' UITableViewCellContentView:0x7f8b32abfeb0.minX == 0   (active, names: '|':Bento.TableViewContainerCell:0x7f8b34055000'(extension in BabylonUI):...' )>     Marker:0x7f8b31576fb0:UIView-Encapsulated-Layout-Left.marker{id: 30491}
    <NSLayoutConstraint:0x7f8b3155f280 UIView:0x7f8b3155ef40.width == 0.2*UIStackView:0x7f8b31584f90.width - 3   (active)>      Marker:0x7f8b3155f280.marker{id: 30263}
    <NSLayoutConstraint:0x7f8b31566e80 'UISV-alignment' UIView:0x7f8b31581680.bottom == UIView:0x7f8b3155ef40.bottom   (active)>        Marker:0x7f8b31566e80:UISV-alignment.marker{id: 30393}
    <NSLayoutConstraint:0x7f8b31567280 'UISV-alignment' UIView:0x7f8b31581680.top == UIView:0x7f8b315818b0.top   (active)>      Marker:0x7f8b31567280:UISV-alignment.marker{id: 30396}
    <NSLayoutConstraint:0x7f8b31567460 'UISV-alignment' UIView:0x7f8b31581680.top == UIView:0x7f8b3155ef40.top   (active)>      Marker:0x7f8b31567460:UISV-alignment.marker{id: 30399}
    <NSLayoutConstraint:0x7f8b315678b0 'UISV-canvas-connection' H:[UIView:0x7f8b3155ef40]-(0)-|   (active, names: '|':UIStackView:0x7f8b31584f90 )>     Marker:0x7f8b315678b0:UISV-canvas-connection.marker{id: 30366}
    <NSLayoutConstraint:0x7f8b31567a60 'UISV-canvas-fit' UIStackView:0x7f8b31584f90.width == 0 priority:49   (active)>      Marker:0x7f8b31567a60:UISV-canvas-fit.posErrorMarker{id: 30368}
    <NSLayoutConstraint:0x7f8b31567ab0 'UISV-fill-equally' _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.width == _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.width   (active)>     Marker:0x7f8b31567ab0:UISV-fill-equally.marker{id: 30381}
    <NSLayoutConstraint:0x7f8b31568790 'UISV-distributing-edge' H:[UIView:0x7f8b31581680]-(0)-[_UIOLAGapGuide:0x7f8b31568650'UISV-distributing']   (active)>        Marker:0x7f8b31568790:UISV-distributing-edge.marker{id: 30370}
    <NSLayoutConstraint:0x7f8b315688a0 'UISV-distributing-edge' _UIOLAGapGuide:0x7f8b31568650'UISV-distributing'.trailing == UIView:0x7f8b315818b0.leading   (active)>      Marker:0x7f8b315688a0:UISV-distributing-edge.marker{id: 30372}
    <NSLayoutConstraint:0x7f8b315691c0 'UISV-spacing' H:[UIView:0x7f8b31581680]-(>=3)-[UIView:0x7f8b315818b0]   (active)>       Marker:0x7f8b315691c0:UISV-spacing.slackMarker{id: 30374}
    <NSLayoutConstraint:0x7f8b31569420 'UISV-distributing-edge' H:[UIView:0x7f8b315818b0]-(0)-[_UIOLAGapGuide:0x7f8b31569320'UISV-distributing']   (active)>        Marker:0x7f8b31569420:UISV-distributing-edge.marker{id: 30377}
    <NSLayoutConstraint:0x7f8b31569530 'UISV-distributing-edge' _UIOLAGapGuide:0x7f8b31569320'UISV-distributing'.trailing == UIView:0x7f8b3155ef40.leading   (active)>      Marker:0x7f8b31569530:UISV-distributing-edge.marker{id: 30379}
    <NSLayoutConstraint:0x7f8b3156b450 'UISV-spacing' H:[UIView:0x7f8b315818b0]-(>=3)-[UIView:0x7f8b3155ef40]   (active)>       Marker:0x7f8b3156b450:UISV-spacing.slackMarker{id: 30385}
    <NSLayoutConstraint:0x7f8b3156b7f0 'UISV-alignment' UIView:0x7f8b31581680.bottom == UIView:0x7f8b315818b0.bottom   (active)>        Marker:0x7f8b3156b7f0:UISV-alignment.marker{id: 30390}
    <NSLayoutConstraint:0x7f8b3156bed0 'UISV-canvas-connection' V:[UIView:0x7f8b31581680]-(0)-|   (active, names: '|':UIStackView:0x7f8b31584f90 )>     Marker:0x7f8b3156bed0:UISV-canvas-connection.marker{id: 30388}
    <NSLayoutConstraint:0x7f8b31576f60 'UIView-Encapsulated-Layout-Height' UITableViewCellContentView:0x7f8b32abfeb0.height == 92.5   (active)>     Marker:0x7f8b31576f60:UIView-Encapsulated-Layout-Height.marker{id: 30496}
    <NSLayoutConstraint:0x7f8b315799e0 'UIView-Encapsulated-Layout-Width' UITableViewCellContentView:0x7f8b32abfeb0.width == 375   (active)>        Marker:0x7f8b315799e0:UIView-Encapsulated-Layout-Width.marker{id: 30493}
    <NSLayoutConstraint:0x7f8b3157d620 'UISV-canvas-connection' UIStackView:0x7f8b31584f90.leading == UIView:0x7f8b31581680.leading   (active)>     Marker:0x7f8b3157d620:UISV-canvas-connection.marker{id: 30364}
    <NSLayoutConstraint:0x7f8b31583800 UIView:0x7f8b3157e7c0.height == 10   (active)>       Marker:0x7f8b31583800.marker{id: 30269}
    <NSLayoutConstraint:0x7f8b31585720 'UISV-canvas-connection' UIStackView:0x7f8b31584f90.top == UIView:0x7f8b31581680.top   (active)>     Marker:0x7f8b31585720:UISV-canvas-connection.marker{id: 30386}
    <NSLayoutConstraint:0x7f8b31586170 UIView:0x7f8b3157e7c0.width == 8   (active)>     Marker:0x7f8b31586170.marker{id: 30267}

  Statistics:
    23 rows. Variable counts:
          1 ->   8
          2 ->   3
          3 ->   3
          4 ->   3
          5 ->   3
          6 ->   1
          7 ->   1
         13 ->   1
    30 cols. Variable counts:
          1 ->   9
          2 ->   5
          3 ->   6
          4 ->   3
          5 ->   2
          6 ->   4
          7 ->   1
}.'
```
</details>

<details>
<summary><tt>*** First throw call stack:</tt></summary>

```
(
    0   CoreFoundation                      0x0000000102ebf6fb __exceptionPreprocess + 331
    1   libobjc.A.dylib                     0x0000000102463ac5 objc_exception_throw + 48
    2   CoreFoundation                      0x0000000102ebf482 +[NSException raise:format:arguments:] + 98
    3   Foundation                          0x0000000101d219f0 -[NSAssertionHandler handleFailureInFunction:file:lineNumber:description:] + 166
    4   Foundation                          0x0000000101f17710 -[NSISEngine removeConstraintWithMarker:] + 1651
    5   Foundation                          0x0000000101f2baec -[NSLayoutConstraint _removeFromEngine:] + 229
    6   UIKitCore                           0x0000000107c8d44d __57-[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:]_block_invoke + 217
    7   Foundation                          0x0000000101f14fa2 -[NSISEngine withBehaviors:performModifications:] + 110
    8   UIKitCore                           0x0000000107c8d33c -[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:] + 217
    9   UIKitCore                           0x0000000107c8d6ab __57-[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:]_block_invoke_2 + 213
    10  UIKitCore                           0x0000000107c8d5af __57-[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:]_block_invoke + 571
    11  Foundation                          0x0000000101f14fa2 -[NSISEngine withBehaviors:performModifications:] + 110
    12  UIKitCore                           0x0000000107c8d33c -[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:] + 217
    13  UIKitCore                           0x0000000107c8d6ab __57-[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:]_block_invoke_2 + 213
    14  UIKitCore                           0x0000000107c8d5af __57-[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:]_block_invoke + 571
    15  Foundation                          0x0000000101f14fa2 -[NSISEngine withBehaviors:performModifications:] + 110
    16  UIKitCore                           0x0000000107c8d33c -[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:] + 217
    17  UIKitCore                           0x0000000107c8d6ab __57-[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:]_block_invoke_2 + 213
    18  UIKitCore                           0x0000000107c8d5af __57-[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:]_block_invoke + 571
    19  Foundation                          0x0000000101f14fa2 -[NSISEngine withBehaviors:performModifications:] + 110
    20  UIKitCore                           0x0000000107c8d33c -[UIView(AdditionalLayoutSupport) _switchToLayoutEngine:] + 217
    21  UIKitCore                           0x0000000107d3f4c7 __45-[UIView(Hierarchy) _postMovedFromSuperview:]_block_invoke + 112
    22  UIKitCore                           0x0000000107d3f3cd -[UIView(Hierarchy) _postMovedFromSuperview:] + 795
    23  UIKitCore                           0x0000000107d3d3ac __UIViewWasRemovedFromSuperview + 172
    24  UIKitCore                           0x0000000107d3ce8d -[UIView(Hierarchy) removeFromSuperview] + 493
    25  Bento                               0x000000011a395416 $s5Bento0A12ReusableViewPAAE09containerC9DidChange4from2toySo6UIViewCSg_AItF + 1094
    26  Bento                               0x000000011a427dbd $s5Bento22TableViewContainerCellC09containedC0So6UIViewCSgvW + 189
    27  Bento                               0x000000011a427fc0 $s5Bento22TableViewContainerCellC09containedC0So6UIViewCSgvs + 288
    28  Bento                               0x000000011a429cc9 $s5Bento22TableViewContainerCellCAA0a8ReusableC0A2aDP09containedC0So6UIViewCSgvsTW + 9
    29  Bento                               0x000000011a3942cf $s5Bento0A12ReusableViewPAAE14unbindIfNeeded33_C920B861C8A0418300F3C0EC0904FFD1LL07removesC0AA13AnyRenderableVSgSb_tF + 847
    30  Bento                               0x000000011a394348 $s5Bento0A12ReusableViewPAAE14unbindIfNeededyyF + 72
    31  Bento                               0x000000011a4290cf $s5Bento22TableViewContainerCellCfD + 79
    32  Bento                               0x000000011a429124 $s5Bento22TableViewContainerCellCfDTo + 36
    33  CoreFoundation                      0x0000000102ded5ff -[__NSArrayM dealloc] + 159
    34  libobjc.A.dylib                     0x000000010247672c _ZN11objc_object17sidetable_releaseEb + 202
    35  UIKitCore                           0x0000000107acfdd2 -[UITableView .cxx_destruct] + 1561
    36  libobjc.A.dylib                     0x0000000102462963 _ZL27object_cxxDestructFromClassP11objc_objectP10objc_class + 127
    37  libobjc.A.dylib                     0x000000010246e4c7 objc_destructInstance + 134
    38  libobjc.A.dylib                     0x000000010246e4f9 object_dispose + 19
    39  UIKitCore                           0x00000001078b285b -[UIResponder dealloc] + 145
    40  UIKitCore                           0x0000000107d26918 -[UIView dealloc] + 1041
    41  UIKitCore                           0x0000000107cca300 -[UIScrollView dealloc] + 900
    42  UIKitCore                           0x0000000107a9735d -[UITableView dealloc] + 473
    43  libsystem_blocks.dylib              0x00000001050d19cd _Block_release + 105
    44  UIKitCore                           0x0000000107d4908a -[UIViewAnimationBlockDelegate .cxx_destruct] + 58
    45  libobjc.A.dylib                     0x0000000102462963 _ZL27object_cxxDestructFromClassP11objc_objectP10objc_class + 127
    46  libobjc.A.dylib                     0x000000010246e4c7 objc_destructInstance + 134
    47  libobjc.A.dylib                     0x000000010246e4f9 object_dispose + 19
    48  libobjc.A.dylib                     0x000000010247672c _ZN11objc_object17sidetable_releaseEb + 202
    49  CoreFoundation                      0x0000000102ebde8e -[__NSDictionaryI dealloc] + 126
    50  libobjc.A.dylib                     0x000000010247672c _ZN11objc_object17sidetable_releaseEb + 202
    51  libobjc.A.dylib                     0x0000000102476e7f _ZN12_GLOBAL__N_119AutoreleasePoolPage3popEPv + 779
    52  CoreFoundation                      0x0000000102edde56 _CFAutoreleasePoolPop + 22
    53  CoreFoundation                      0x0000000102e20f5e __CFRunLoopRun + 2318
    54  CoreFoundation                      0x0000000102e20302 CFRunLoopRunSpecific + 626
    55  XCTest                              0x0000000102347f24 -[XCTWaiter waitForExpectations:timeout:enforceOrder:] + 996
    56  XCTest                              0x000000010234adb1 +[XCTWaiter waitForExpectations:timeout:enforceOrder:] + 104
    57  XCTest                              0x000000010236884d -[XCTTestRunSession runTestsAndReturnError:] + 895
    58  XCTest                              0x00000001022d541c -[XCTestDriver runTestsAndReturnError:] + 446
    59  XCTest                              0x000000010235a56f _XCTestMain + 2333
    60  xctest                              0x00000001019d8cb9 main + 263
    61  libdyld.dylib                       0x000000010504b541 start + 1
    62  ???                                 0x0000000000000005 0x0 + 5
)
```
</details>

`libc++abi.dylib: terminating with uncaught exception of type NSException`